### ISSUE TITLE
Add SQLClient.bulk_insert()

### ISSUE THREAD

**Issue by dgilland** [dgilland/sqlservice#6]
This would be a convenience method with the following usage:

```python
db.bulk_insert(SomeModel, list_of_dicts)
```

which would be equivalent to doing:

```python
#db.execute(SomeModel.__table__.insert(), list_of_dicts)  # scratch this in favor of next line
db.execute(SomeModel.__table__.insert().values(list_of_dicts))
```

EDIT: Updated equivalent example based on suggestion by @jeffwidman.

**Comment by jeffwidman** [comment]
For speed, you probably want this to do:

    db.execute(SomeModel.__table__.insert().values(list_of_dicts)

See comments on this SO answer: http://stackoverflow.com/a/36879384/770425

**Comment by dgilland** [comment]
@jeffwidman Thanks! I did not know about that alternative syntax. From what I've just been reading, that does look like the way I'd want to go.

**Comment by dgilland** [comment]
Implemented in 79306b2dc98410ce1f6dfd004aa3052ad62397ef.
